### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#67071]
#### VFS team name
Authenticated Experience

#### VFS product name
Profile

#### VFS feature name
Notification settings > Editing as a subtask

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
When in the editing subtask, clicking "Cancel" appears to be purely navigational -- you're taken back to the previous page and no data is saved or changed. Buttons shouldn't be used for navigation.

**Edit:** As discussed in the staging review meeting, this is not an issue and can be disregarded. The reasoning here around the "are you sure" modal when changes have been started makes sense, this is all good as a button.

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Cancel should be a link! I believe it was planned to be a link at Midpoint, so please let me know if there's something that came up after we last spoke that changed and made a button more appropriate here.

**Edit:** See note above. No change is needed.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by mtcA6** [comment]
@shiragoodman We had cancel as a button, in design intent it [was recommended we change it to a link in the must feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/59155), then we got to midpoint where we had this as a link and it was requested we shorten the language and it was recommended we change back to a button. Since it is a single destructive action, we followed the recommendation and added a modal when a user hits cancel which then had the team torn between a button or a link and felt it could go either way. The midpoint tickets don't mention making it a button but I'm fairly confident we spent time on the midpoint call talking about it. 

It's only navigational if a user hasn't entered any data into the field. 

@adamwhitlock1 do you recall this the same way I do? I feel we've flip-flopped on this a couple of times now.

**Comment by adamwhitlock1** [comment]
Yea we did go back and forth on this, and in the staging review the usage of a button was discussed and everyone was cool with it being a button.

I believe this ticket can be closed or disregarded as it says:

> Edit: As discussed in the staging review meeting, this is not an issue and can be disregarded. The reasoning here around the "are you sure" modal when changes have been started makes sense, this is all good as a button.

**Comment by mtcA6** [comment]
oooops, im sorry i didn't read that part 🤦
